common/tools/dev-tool/src/util/prettier.ts

```diff
@@ -0,0 +1,16 @@
+import * as prettier from "prettier";
+
+const prettierOptions: prettier.Options = {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  ...(require("../../../eslint-plugin-azure-sdk/prettier.json") as prettier.Options),
```

Comment:
Do we want to use require here or should we just be using fs to read the file?

Reply:
I guess this is just how I did it previously in the first iterations of the dev-tool sample processor. It's just simpler but I'm happy to do a `readFileSync` into `JSON.parse` instead. Without top-level async in all our node targets I can't easily use async file IO in the module body, so that's why I did it this way.